src/controllers/UserController.js

```diff
@@ -0,0 +1,43 @@
+const User = require('../models').User;
+const { getHashedPassword } = require('../helpers/helpers');
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt');
+
+module.exports = {
+  async register(req, res) {
+    try {
+      const { username, password } = req.body;
+      const alreadyExistsUser = await User.findOne({ where: { username } });
+      if (alreadyExistsUser) {
+        return res.status(400).send({ error: 'User with this username already exists' });
+      }
+      const hashedPassword = await getHashedPassword(password);
+      const newUser = await User.create({ username, password: hashedPassword });
+      await newUser.save();
+      return res.status(200).send(newUser);
+    } catch (error) {
+      res.status(500).send(error);
+    }
+  },
+  async login(req, res) {
+    try {
+      const { username, password } = req.body;
+
+      const userWithUsername = await User.findOne({ where: { username } });
```

Comment:
Почему у переменной такое название ? Почему не user
```suggestion
      const userWithUsername = await User.findOne({ where: { username } });
```

Reply:
Потому что, это переменная, для того чтобы проверить, есть ли такой юзернейм в системе. Чтобы не было дублирование.